src/app/item-popup/PlugTooltip.tsx

```diff
@@ -49,21 +51,64 @@ export default function PlugTooltip({
       )
     : [];
 
+  const stats: { [statHash: string]: number } = {};
+
+  for (const statHash of visibleStats) {
+    const value = plug.stats?.[parseInt(statHash, 10)];
+    if (typeof value === 'number') {
```

Comment:
When is value not a number?

Reply:
I think that is hangover from the first pass, it can just be undefined I think.